lte/gateway/c/session_manager/LocalEnforcer.cpp

```diff
@@ -1768,52 +1760,6 @@ bool LocalEnforcer::session_with_apn_exists(
   return false;
 }
 
-bool LocalEnforcer::is_session_active(
```

Comment:
are those functions deleted here not needed anymore?

Reply:
Yeah removed, they were only used once in the session recycling logic. And when I moved things around, it was much more convenient to just unroll these functions.